Kajabi/sage-lib
# Panel Controls(Rails) - update pagination and bulk actions selection

## Description

- [x] - add `kaminari` gem so that we can use backend `ActiveRecord` functionality to product demos with `SagePagination` in isolation or within `SagePanelControls`
- [x] - add `SagePagination` to panel controls
- [x] - updated `bulkactions` conditional to not include pagination because pagination can exist without `bulkactions`
- [x] - updated `SagePagination` docs due to new props
- [x] - updated `SagePanelControls` docs to account for `SagePagination` use enabled

### Screenshots

No visual changes but added demo variations
![Screen Shot 2021-03-24 at 1 18 53 PM](https://user-images.githubusercontent.com/1241836/112363553-a3c3ad00-8ca3-11eb-9d27-1a4d347be5e9.png)


## Test notes


### Steps for testing
pending
- Visit rails panel controls page: http://localhost:4000/pages/object/panel_controls
- Notice the updated variations of the panel controls

### QA Steps
#380 (Low)  There should be no change as all work is rails based and any styles are behind classes that only exist on the rails variation
- [ ] - Contacts page panel controls(React)
- [ ] - no existing rails implementation
